satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -368,6 +369,7 @@ model api_key (
     field  head        blob
     field  name        text       (updatable)
     field  secret      blob
+	field  partner_id  blob       (nullable)
```

Comment:
all the formatting -- I'll stop commenting on it though

Reply:
Oops...how did I miss that... looked good on my vs code... will fix this...